app/config.py

```diff
@@ -244,6 +244,16 @@ def sl_getenv(env_var: str, default_factory: Callable = None):
 FACEBOOK_CLIENT_ID = os.environ.get("FACEBOOK_CLIENT_ID")
 FACEBOOK_CLIENT_SECRET = os.environ.get("FACEBOOK_CLIENT_SECRET")
 
+CONNECT_WITH_OIDC = "CONNECT_WITH_OIDC" in os.environ
```

Comment:
can we add "ENABLE_" prefix to "CONNECT_WITH_OIDC" to make it more explicit that it's a boolean?

Reply:
Or even use directly if `OIRC_CLIENT_ID` is set to anything

Reply:
i based it directly on `OIDC_CLIENT_ID`.